proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -604,3 +604,41 @@ message ForceUnloadTaskQueuePartitionRequest {
 message ForceUnloadTaskQueuePartitionResponse {
   bool was_loaded = 1;
 }
+
+// StartAdminBatchOperationRequest starts an admin batch operation.
+message StartAdminBatchOperationRequest {
+  // Namespace that contains the batch operation.
+  string namespace = 1;
+
+  // Visibility query defines the group of workflows to apply the batch operation.
+  // This field and `executions` are mutually exclusive.
```

Comment:
I think we will want to support a list of executions as well, that can be quite useful during incident mitigation, but can be done as a follow up item.